visualizations/fan_chart/webviz_fan_chart/__init__.py

```diff
@@ -170,21 +172,28 @@ def __init__(
             self,
             data,
             observations=None,
-            logx=False,
-            logy=False,
+            logx=False, logy=False,
+            xrange=None, yrange=None,
```

Comment:
According to [plotly documentation on `range`](https://plot.ly/javascript/reference/#layout-yaxis-range), if the axis is logarithmic, the values in `range` should be the values after taking `log`.

I think we should do that `log`-transformation for the user. I.e. add the transformation of `[x,y]range` within the corresponding `PageElement` (given that `log[x,y] = True`)